passenger-service/src/main/java/com/example/passengerservice/service/PassengerService.java

```diff
@@ -0,0 +1,11 @@
+package com.example.passengerservice.service;
+
+import com.example.passengerservice.dto.request.PassengerRequest;
+import com.example.passengerservice.dto.response.PassengerResponse;
+
+public interface PassengerService {
```

Comment:
Minor: in small apps there is no need to create interfaces for services until you don't have hierachy or several implementations